taca/utils/misc.py

```diff
@@ -109,6 +110,25 @@ def days_old(date, date_format="%y%m%d"):
         return None
     return time_dif.days
 
+
+def to_seconds(days=None, hours=None):
+    """ Convert given day/hours to seconds and return
+
+        :param int days: days to be converted
+        :param int hours: hours to be converted
+    """
+    #only either days or hours should be speciefied, but atleast one should be specified
+    if days and hours:
+        raise SystemExit('Both "days" and "hours" were given, use only either of them')
+    elif not days and not hours:
```

Comment:
Here there might be an unexpected behaviour is one of `days` or `hours` is set to `0`.
In such a case they will be evaluated in the same way as `None` is evaluated.  Here the question is how should the script behave if cleaning time is set to 0, i.e., delete/clean everything. I think we should never allow such a thing, things need to be at least one hour old.... so a way is to set default to 0 for both arguments.